apm-sniffer/apm-sdk-plugin/kafka-plugin/src/main/java/org/apache/skywalking/apm/plugin/kafka/ProducerConstructorMapInterceptor.java

```diff
@@ -16,14 +16,21 @@
  *
  */
 
-package org.apache.skywalking.apm.plugin.kafka.v1;
+package org.apache.skywalking.apm.plugin.kafka;
 
 import org.apache.skywalking.apm.agent.core.plugin.interceptor.enhance.EnhancedInstance;
 import org.apache.skywalking.apm.agent.core.plugin.interceptor.enhance.InstanceConstructorInterceptor;
+import org.apache.skywalking.apm.util.StringUtil;
 
-public class ProducerRecordConstructorInterceptor implements InstanceConstructorInterceptor {
-    @Override public void onConstruct(EnhancedInstance objInst, Object[] allArguments) {
-        String topic = (String)allArguments[0];
-        objInst.setSkyWalkingDynamicField(topic);
+import java.util.Map;
+
+/**
+ * @author stalary
+ */
+public class ProducerConstructorMapInterceptor implements InstanceConstructorInterceptor {
+    @Override
+    public void onConstruct(EnhancedInstance objInst, Object[] allArguments) {
+        Map<String, Object> config = (Map<String, Object>) allArguments[0];
+        objInst.setSkyWalkingDynamicField(StringUtil.join(';', ((String) config.get("bootstrap.servers")).split(",")));
```

Comment:
The `bootstrap.servers` is just for tagging, right? Do we really need to spilt it with `,` and join them with another delimiter `;` ? What about just setting them **AS IS**

Reply:
Here is a compatibility operation. Before 2.1.0 you get a List and then a string, for uniform display

Reply:
I didn't change the `;` used in previous versions for consistency

Reply:
What I meant is the bootstrap.servers is separated by `,` in Kafka, here we can just keep it as is, and when you get a `List`, join them with `,` instead of `;`. Anyway, it's trivial and it's ok to me, thanks